### PR TITLE
VCA update email error handling

### PR DESCRIPTION
**Pull request by krystophv**
Bring error handling for update profile in line with create custodial account for the edit/create screen.

### CODE REVIEW

**Comment by krystophv** [comment]
Some updates necessary for error handling. ping @jh-bate for review.

**Comment by krystophv** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;XfHSKoKc&#x2F;12-ux-0-dev-1-tst-1-uploader-add-an-email-address-to-a-custodial-account-after-it-s-been-created"> &lt;ux:0 dev:1 tst:1&gt; [uploader] Add an email address to a custodial account after it&#39;s been created</a>

File: lib/components/ClinicUserEdit.js [hunk 1]
```diff
@@ -68,8 +68,10 @@ var MONTHS = [
 
 var ClinicUserEdit = React.createClass({
   propTypes: {
-    errorMessage: React.PropTypes.string,
-    errorDismissed: React.PropTypes.bool.isRequired,
+    createErrorMessage: React.PropTypes.string,
```
Comment (jh-bate) [inline lib/components/ClinicUserEdit.js]: to keep the naming consistent maybe these should be `createProfileErrorMessage` if the other is `updateProfileErrorMessage`. Thoughts?

File: lib/pages/App.js [hunk 2]
```diff
@@ -247,9 +247,12 @@ export default class App extends Component {
             createUser={this.props.async.createCustodialAccount}
             updateUser={this.props.async.clickEditUserNext}
             cancelEdit={_.partial(this.handleClickChangePerson, {metric: {eventName: metrics.CLINIC_ADD_CANCEL}})}
-            errorMessage={this.props.createCustodialAccountErrorMessage}
-            errorDismissed={this.props.createCustodialAccountErrorDismissed}
+            createErrorMessage={this.props.createCustodialAccountErrorMessage}
```
Comment (jh-bate) [inline lib/pages/App.js]: `createErrorMessage` => `createCustodialAccountErrorMessage` which give a better indication of what we are creating and matches `updateProfileErrorMessage={this.props.updateProfileErrorMessage}`  better ...

File: lib/redux/utils/errors.js [hunk 3]
```diff
@@ -67,6 +67,8 @@ export function getLoginErrorMessage(status) {
 
 export function getUpdateProfileErrorMessage(status) {
   switch (status) {
+    case 400:
+      return 'Something looks funky, make sure this account info is correct.';
```
Comment (jh-bate) [inline lib/redux/utils/errors.js]: does the language need to be vetted?

File: test/browser/redux/actions/async.test.js [hunk 4]
```diff
@@ -2090,6 +2090,49 @@ describe('Asynchronous Actions', () => {
         expect(actions).to.deep.equal(expectedActions);
       });
     });
+    describe('update profile failure, unauthorized', () => {
```
Comment (jh-bate) [inline test/browser/redux/actions/async.test.js]: do we have a success test also?

File: lib/redux/utils/errors.js (hunk 3 above)
Comment (krystophv) [inline lib/redux/utils/errors.js]: It's the same error message as https://github.com/tidepool-org/chrome-uploader/pull/317/files#diff-ed50da20752781857ba122369a967365R84 which is presenting the same error, so I think we're good.

File: test/browser/redux/actions/async.test.js (hunk 4 above)
Comment (krystophv) [inline test/browser/redux/actions/async.test.js]: A couple, actually: https://github.com/tidepool-org/chrome-uploader/pull/317/files#diff-be6f22c962702fffabcf5837ef8f09bdR1938 and https://github.com/tidepool-org/chrome-uploader/pull/317/files#diff-be6f22c962702fffabcf5837ef8f09bdR2003

**Comment by jh-bate** [comment]
LGTM

**Pull request merged**